setup.py

```diff
@@ -7,11 +7,11 @@
 if sys.version_info[0] == 2:
     install_requires = ["enum34", "typing"]
 
-with open("README.md") as f:
-    readme = f.read()
+with open("README.md", "rb") as f:
+    readme = f.read().decode("utf-8")
```

Comment:
```suggestion
with open("README.md", encoding="utf-8") as f:
    readme = f.read()
```
maybe we can sole it by [the officially recommended way](https://packaging.python.org/guides/making-a-pypi-friendly-readme/).

Reply:
Python 2 seems to not support encoding parameter, so I avoided it: https://docs.python.org/2.7/library/functions.html#open

But I guess I may try it, and let's see whether the CI will complain.

Reply:
It failed in CI:
```
TypeError: 'encoding' is an invalid keyword argument for this function
```

https://travis-ci.org/github/tomtom-international/openlr-python/jobs/689304033